_posts/2020/2020-04-23-instagram-blog-post.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: 5 tips to keep your company Instagram channel alive / 5 tips on how to manage your company Instagram channel
+date: 2020-06-04
+header_image: public/instagram-company-account.jpg
+header_position: center
+header_overlay: true
+category: techcomm-and-socialmedia
+tags: ["social media", "Instagram", "content"]
+authors: ["Aroso"]
+about_authors: ["amahaiuddin"]
+---
+
+It is called a picture is worth a thousand words - Instagram is the perfect social media platform for sharing visuals around the world.
+We started our Instagram channel[@life_at_epages](https://www.instagram.com/life_at_epages/){:target="_blank"} in 2019 to present our company culture and express thus who we are.
+But managing a company Instagram channel and coordinating content involves some important aspects to consider.
+In the following I will explain which actions we take in order to keep our company Instagram channel alive.
+
+## Create Instagram How-to guide
+
+Usually, an Instagram company channel is not run by a single person but by a team consisting of two or more employees.
+In order to develop and create consistent content for our Instagram channel, we created an Instagram How-to guide.
+This guide shows the employees an overview of how to use Instagram at ePages and is continuously being updated.
+
+- **Aim of Instagram channel**:
+Write down what you want to reach with the Instagram channel and in which areas the channel should support the company.
+Should it support you more in promoting your company product, should it support you in reaching out to potential hires, or should it support you more in gaining traffic on your company blog.
+
+- **Target group**:
+Define your target group: Write down the groups you would like to reach with your Instagram channel as well as groups which do not belong to your target group.
+This makes it more clear for the employees which group of persons you would like to reach out to.
+Thus, you can orient your content towards your defined target group.
+
+- **Content ideas**:
+List in a table content ideas that show which kind of content is suitable for your Instagram channel, e.g. internal & external events, employee portraits, or team activities.
+Furthermore, you can specify the content ideas by describing them.
+For example, write down which external events you want to visualize - is it only conferences, or fairs as well?
```

Comment:
Idea for not using "write down" again. 

Here’s an example: Which external events would you like to visualize - is it only conferences, or fairs as well?

(By the way, I would remove the next sentence.... otherwise it might be too close to the content plan?)